Changelog.md

```diff
@@ -1,5 +1,8 @@
 # Spectrometer Changelog
 
+## v2.15.22
+- The output from `analyze` command includes the Scan URL with branch and revisionID ([#378](https://github.com/fossas/spectrometer/pull/378))
```

Comment:
Please clarify that this is when running with `--experimental-enable-monorepo`; this change doesn't alter the standard `fossa analyze` command (which already rendered the result correctly)

Reply:
Gotcha, thanks!